app/medInria/medLogger.cpp

```diff
@@ -132,3 +137,34 @@ void medLogger::createTeeStream(std::ostream* targetStream)
     targetStream->rdbuf(d->teeStreams.last()->rdbuf());
 }
 
+void medLogger::truncateLogFileIfHeavy()
+{
+    qint64 filesize = QFileInfo(dtkLogPath(qApp)).size();
+
+    // Over 5Mo, the file is truncated from the beginning (old lines are discarded)
+    if (filesize > d->maxLogSize)
+    {
+        QString path = dtkLogPath(qApp);
+
+        std::ifstream fin;
+        fin.open(path.toUtf8());
+        fin.seekg(filesize-d->minLogSize); // file is going to be cut to 1Mo
```

Comment:
You should replace "1Mo" in the comment by ```minLogSize```
Also there seems to be a syntax error here : ```filesize-d->minLogSize```